public/views/index.html

```diff
@@ -176,25 +192,29 @@
         </div>
       </div>
       <div class="preloader__text preloader__text--fail">
-        <p>
-          <strong>If you're seeing this Grafana has failed to load its application files</strong>
-          <br />
-          <br />
-        </p>
-        <p>
-          1. This could be caused by your reverse proxy settings.<br /><br />
-          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath. If not
-          using a reverse proxy make sure to set serve_from_sub_path to true.<br />
-          <br />
-          3. If you have a local dev build make sure you build frontend using: yarn start, or yarn build<br />
-          <br />
-          4. Sometimes restarting grafana-server can help<br />
-          <br />
-          5. Check if you are using a non-supported browser. For more information, refer to the list of
-          <a href="https://grafana.com/docs/grafana/latest/installation/requirements/#supported-web-browsers">
-            supported browsers</a
-          >.
-        </p>
+        <main>
+          <h1>If you're seeing this Grafana has failed to load its application files</h1>
+          <p>
+            1. This could be caused by your reverse proxy settings.
```

Comment:
I am having trouble reading beyond point 1 with VoiceOver. Maybe we should make this into and `ol` instead?